arch/arm64/boot/dts/qcom/sdm450-samsung-gta2xl-r5.dts

```diff
@@ -0,0 +1,405 @@
+// Planes are manufactured by boeing.
+// I have no idea what I am doing.
+// - NSG650
+
+/dts-v1/;
+
+#include "sdm450.dtsi"
+#include "msm8953.dtsi"
+#include "pm8953.dtsi"
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Samsung Tab A2 XL Rev05";
+	compatible = "samsung,gta2xl", "qcom,sdm450";
+	chassis-type = "tablet";
+	qcom,msm-id = <351 0x0>;
+	qcom,board-id = <8 5>;
+
+	aliases {
+		mmc0 = &sdhc_1;
+		mmc1 = &sdhc_2;
+		serial0 = &uart_0;
+	};
+
+	chosen {
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges;
+
+		framebuffer0: framebuffer@90001000 {
+			compatible = "simple-framebuffer";
+			reg = <0 0x90001000 0 (1920 * 1200 * 3)>;
+
+			width = <1200>;
+			height = <1920>;
+			stride = <(1200 * 3)>;
+			format = "r8g8b8";
+
+			power-domains = <&gcc MDSS_GDSC>;
+
+			clocks = <&gcc GCC_MDSS_AHB_CLK>,
+				 <&gcc GCC_MDSS_AXI_CLK>,
+				 <&gcc GCC_MDSS_VSYNC_CLK>,
+				 <&gcc GCC_MDSS_MDP_CLK>,
+				 <&gcc GCC_MDSS_BYTE0_CLK>,
+				 <&gcc GCC_MDSS_PCLK0_CLK>,
+				 <&gcc GCC_MDSS_ESC0_CLK>;
+		};
+	};
+
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		volume-up {
+			label = "Volume Up";
+			gpios = <&tlmm 85 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_VOLUMEUP>;
+		};
+	};
+
+	reserved-memory {
+		memory@82800000 {
+			reg = <0x0 0x82800000 0x0 0x2800000>;
+			no-map;
+		};
```

Comment:
These reserved memory ranges should have names. Like wcnss_fw_mem, adsp_fw_mem etc.